### DISCUSSION TITLE
Explanation for log "level" numbers

### DISCUSSION THREAD

**Discussion by ronaldploeger** [renovatebot/renovate#8669] [Request Help] [answered]
When using LOG_LEVEL=debug and LOG_FORMAT=json renovate uses numbers like 20 and 30 for the "level" field, e.g.

`{"name":"renovate","hostname":"renovate-manual-kh5-lx467","pid":7,"level":20,"logContext":"Gl_rXIr_I","repository":"xxx/xxx","msg":"Removing any stale branches","time":"2021-02-12T17:08:04.024Z","v":0}`

Is there anywhere I can find an explanation which number means what? Thanks.

**Comment by rarkins** [comment, marked as answer]
It's controlled by the Bunyan logging library. 10=trace, 20=debug, 30=info, 40=warn, 50=error, 60=fatal. PR welcome if you'd like to update the logging docs

**Reply by ronaldploeger** [reply]
Thank you @rarkins. I will take a look at the docs and create a PR.
